### DISCUSSION TITLE
Accessing darkMode options in .md docs file

### DISCUSSION THREAD

**Discussion by san-sei** [thuliteio/doks#668] [Q&A] [answered]
@h-enk  I am wondering if is it possible to get access to the current mode (dark or light) value in the .md file of document files (introduction.md for example) to reference two different images/icons when needed.

**Comment by h-enk** [comment]
No, not out of the box. You would need some JavaScript for that — see also [darkmode-init.js](https://github.com/h-enk/doks/blob/master/assets/js/darkmode-init.js) + [darkmode.js](https://github.com/h-enk/doks/blob/master/assets/js/darkmode.js)

**Reply by h-enk** [reply]
Actually, you could just check for the `data-dark-mode` attribute set on the document element (the logic is already present). And, next reference it via CSS (that's how I did it for some of the sponsor logos on getdoks.org — see also https://github.com/h-enk/getdoks.org/blob/master/layouts/partials/main/doks-sponsors.html#L20). In a `.md` file you would then have to add HTML.

**Comment by san-sei** [comment, marked as answer]
Adding class names `class="logo-light"` OR `class="logo-dark d-none"` as you mentioned worked.
